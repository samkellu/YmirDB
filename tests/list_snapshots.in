set a 123
snapshot
snapshot
snapshot
list snapshots
checkout 1
snapshot
list snapshots
drop 4
list snapshots
bye
