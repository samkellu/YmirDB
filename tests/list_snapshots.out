> ok

> saved as snapshot 1

> saved as snapshot 2

> saved as snapshot 3

> 3
2
1

> ok

> saved as snapshot 4

> 4
3
2
1

> ok

> 3
2
1

> bye
